README.md

```diff
@@ -49,36 +63,157 @@ const { MediaContextProvider, Media, createMediaStyle } = createMedia({
 })
 
 const App = () => (
-  <>
-    <Style>{createMediaStyle()}</Style>
+  <MediaContextProvider>
+    <Media at='sm'>
+      <MobileApp />
+    </Media>
+    <Media at='md'>
+      <TabletApp />
+    </Media>
+    <Media greaterThanOrEqual='lg'>
+      <DesktopApp />
+    </Media>
+  </MediaContextProvider>
+)
+
+ReactDOM.render(<App />, document.getElementById("react"))
+```
+
+## Server-side Rendering (SSR) Usage
+
+The first important thing to note is that when server-rendering with
+`@artsy/fresnel`, all breakpoints get rendered by the server. Each `Media`
+component is wrapped by plain CSS that will only show that breakpoint if it
+matches the user's current browser size. This means that the client can
+accurately start rendering the HTML/CSS while it receives the markup, which is
+long before the React application has booted. This improves perceived
+performance for end-users.
+
+Why not just render the one that the current device needs? A couple of reasons.
```

Comment:
There doesn't appear to be a “couple of reasons” listed, so perhaps remove that?